frameworks/helloworld/tests/test_sanity.py

```diff
@@ -217,15 +269,13 @@ def test_config_cli():
     configs = json.loads(stdout)
     assert len(configs) >= 1  # refrain from breaking this test if earlier tests did a config update
 
-    assert (
-        0
-        == sdk_cmd.svc_cli(
-            config.PACKAGE_NAME,
-            foldered_name,
-            "debug config show {}".format(configs[0]),
-            print_output=False,  # noisy output
-        )[0]
+    rc, _, _ = sdk_cmd.svc_cli(
+        config.PACKAGE_NAME,
+        foldered_name,
+        "debug config show {}".format(configs[0]),
+        print_output=False,  # noisy output
     )
+    assert rc == 0
```

Comment:
instead of `assert rc == 0` here, how about passing `check=True` on previous call to `svc_cli`?

Reply:
same comment for the rest of the similar occurrences in this file.

Reply:
on a closer look, it looks like `check=True` may cause the logging to be skipped. One alternative is to do the check ourself (instead of passing it to `subprocess.run` [here](https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_cmd.py#L199)) like 

```
def _run_cmd(cmd, print_output, check, timeout_seconds=None):
..
..
..
..
  if check:
    assert not result.returncode, stdout + stderr

```
but this may be out of scope and i leave this to you.

Reply:
I did not want to functionally change the code but just make it clearer that we are checking the `rc` value. This also made the formatting a bit cleaner. Switching to `check=True` is out of scope for this PR.